### ISSUE TITLE
Working example for loading pre-trained checkpoints

### ISSUE THREAD

**Issue by tyler-hayes** [jacobswan1/SEED#3]
Hello and thanks for your interesting work!

Could you provide an example script to load in the pre-trained checkpoints? I tried using the loading code from `main_lincls.py` but have not been able to successfully load in the `*.tar` checkpoint files. Thanks in advance!

**Comment by jacobswan1** [comment]
Hi @tyler-hayes, thanks for being interested in the work!

It seems that I can successfully load the *.tar checkpoint in my environment by:

```
import torch

checkpoint = torch.load('resnet18_distill_resnet50-moco-v2-checkpoint_0199.pth.tar', `map_location="cpu")
```

Probably it's because of a different PyTorch version you are using?

**Comment by tyler-hayes** [comment]
Hi @jacobswan1, thanks for your response!

Using `torch.load` successfully loads the checkpoint. However the checkpoint has more parameters than the model I was loading it into. I solved this by only loading parameters with the prefix `module.encoder_q` into the model. Is this correct?

**Comment by jacobswan1** [comment]
@tyler-hayes, Oh yes the encoder_q is the student model. I guess this is because the checkpoint contains the teacher parameters as well. One way to check if parameters are loaded 100% successfully is to use `load_state_dict strict=true` and load the `encoder_q`. If all parameters match and are not missed then it is definitely correct.
